docker/image_test.go

```diff
@@ -37,3 +41,15 @@ func TestParseImageFromName(t *testing.T) {
 		}
 	}
 }
+
+func TestParseImageFromRepoAndSha(t *testing.T) {
+	registries := []string{"", "quay.io"}
```

Comment:
ah, good catch. empty strings will be ignored and `index.docker.io` will be converted to an empty string. I need to add `index.docker.io` to test that behavior. thanks!